app/src/main/res/values-fr/strings.xml

```diff
@@ -8,11 +8,11 @@
 <resources>
 
     <!-- Name der App -->
-    <string name="app_name">"Next Step"</string>
+    <string name="app_name">"Protetti"</string>
```

Comment:
Protetti? That's not french

Reply:
This is the name given to the Italian version (it'a kind of branding). Actually we need to externalize this kind of information because of a few i18n issues.
For example the toll free number to call in order to ask for information about the Covid-19 is different depending the Nation and maybe the user's language (e.g. I'm Italian, I live in France and my phone user interface in in English: what is the expected behaviour of the application if I use it in Germany?). In the NoiApp we are aware of this and we are thinking about a solution.

Reply:
By the way "Protetti" means "Protected"